docs/DeveloperGuide.md

```diff
@@ -377,149 +329,60 @@ The following activity diagram summarizes what happens when a user executes a ne
-`ReadOnlyIngredientBook` interface and offers methods to view the ingredients' levels from the application's 
-`ingredientBook`. Particularly, it implements the following operation:
-  
-  * `IngredientBook#findIngredientByName(IngredientName ingredientName)` — Returns the ingredient with the target 
-  ingredient name. 
-  
-This operation is exposed in the `Model` interface as `Model#findIngredientByName(IngredientName ingredientName)`.
-  
-Given below is an example usage scenario and how the view a single ingredient's level mechanism behaves at each step.
-  
-Step 1. The user launches the application. If the storage file for the ingredient book is empty, `IngredientBook` will 
-be initialized with the five pre-defined ingredients, namely `Milk`, `Pearl`, `Boba`, `Oolong Tea` and `Brown Suagr`, 
-with an amount of 0 for all. If the storage file for the ingredient book is not empty, `IngredientBook` will read the  
-data from the storage file.
-  
-Step 2. The user executes `i-view-single i/Milk` to view milk's current level. The `i-view-single i/Milk` command is
-parsed by `IngredientViewSingleCommandParser` which parses the ingredient to get the ingredient name and 
-returns an  `IngredientViewSingleCommand`. Logic executes the `IngredientViewSingleCommand` and calls 
-`Model#findIngredientByName(IngredientName ingredientName)`, which returns the ingredient with the ingredient name
-entered by the user.
-  
-The following activity diagram shows how the view a single ingredient level operation works:
-![View a Single Ingredient Activity Diagram](images/IngredientViewSingleActivityDiagram.png) 
-
-#### Design consideration:
-  
-##### Aspect: How find the ingredient's level executes
-  
-  * **Alternative 1 (current choice):** Obtain the ingredient name of the ingredient entered by the user, and use the
-  ingredient name to find the ingredient by looping through the ingredient list.
-    * Pros: Code is more readable.
-    * Cons: Every execution of the command will require one to access the name of the ingredient and loop through 
-    the list once, which may increase the time required for the operation. 
-    
-  * **Alternative 2:** Map the ingredient entered by the user to a index which corresponds to the index of the 
-  ingredient in the list, then find the ingredient using the index.
-    * Pros: Do not require looping through the list every time `IngredientViewSingleCommand` executes.
-    * Cons: Code may be less readable.
-    
 ## \[Completed\] Reset all ingredients' levels feature
 
-Reset all ingredients' levels feature allows the user to reset all the ingredient levels to zero. It helps the user to
+tCheck allows the user to reset all the ingredient levels to zero. It helps the user to
 remove data that are no longer needed. The command is:
   
 * `i-reset-all` - Resets the ingredients' levels of all ingredients to zero.
   
 #### Completed Implementation
```

Comment:
Change the subheader to just "Implementation"

Reply:
Okay, thanks!